site/src/sections/hero.style.js

```diff
@@ -0,0 +1,129 @@
+import styled from "styled-components";
+
+const HeroStyleWrapper = styled.div`
+    height: 53rem;
+    width: 100%;
+    background-color: #00B39F;
+    color: #FFF;
+    font-family: 'Qanelas Soft', 'Open Sans', sans-serif;
+    display: flex;
+    postion: relative;
+    overflow: hidden;
+    
+    .content {
+        position: absolute;
+        display: flex;
+        justify-content: space-between;
+        margin-inline: 10%;
+        margin-top: 6%;
+        @media(max-width: 950px) {
+            flex-direction: column;
+            align-items: center;
+            margin-inline: 0;
+            margin-top: 0%;
+            gap: 1rem;
+        }
+    }
+
+    .info {
+        width: 50%;
+        display: flex;
+        flex-direction: column;
+        gap: 1em;
+        margin-top: 4.5rem;
+        @media(max-width: 950px) {
+            width: 90%;
+        }
+
```

Comment:
Maybe we could remove these extra lines in the css ?

Reply:
Hi @legendarykamal,
Thank you for the feedback, I can remove these in the css, but I think it will look strange on smaller screen sizes.